.pylintdict

```diff
@@ -101,11 +108,13 @@ mcrz
 misclassified
 mixin
 monte
+modelling
```

Comment:
I think you should be able to use `modeling` with one `l` which is more the American way - the text is done with US English spell checker as the docs are all done in US English.

Reply:
Can you also delete this addition from the custom dictionary so the US English spelling is the only one that will be allowed.